Dockerfile

```diff
@@ -62,7 +62,8 @@ CMD ["run", "-m", "unittest", "discover", "-s", "tests", "-p", "test_*.py"]
 ## Security Tests
 FROM ci AS test_security
 ENTRYPOINT find . -type f -name '*.py' -exec bandit {} + && \
-  trufflehog --regex --entropy=False file:///usr/src/app/ --exclude_paths .truffleHog-exclude.txt
+  trufflehog --regex --entropy=False file:///usr/src/app/ --exclude_paths .truffleHog-exclude.txt && \
```

Comment:
NIT: I have recently moved to the following approach which makes it easier to remove/change lines independently:
```bash
ENTRYPOINT echo "line 1" \
  && echo "line 2" \
  && echo "line 3"
```